schedule/yast/rmt_server_install/rmt_server_install.yaml

```diff
@@ -0,0 +1,48 @@
+---
+name: rmt_server_install
+description: |
+  Perform an installation enabling some modules and selecting some
+  patterns.This test suite always registers to have access to some modules.
+vars:
+  YUI_REST_API: 1
+schedule:
+  - installation/bootloader_start
+  - installation/setup_libyui
+  - installation/product_selection/install_SLES
+  - installation/licensing/accept_license
+  - installation/registration/register_via_rmt
+  - installation/module_registration/register_nonconflicting_modules
+  - installation/add_on_product/skip_install_addons
+  - installation/system_role/accept_selected_role_SLES_with_GNOME
+  - installation/partitioning/select_guided_setup
```

Comment:
we can remove all these test modules until `accept_proposed_layout` we don't need to change defaults or pass through the guided setup.

Reply:
not inclusive, you still need to accept the proposed layout :)